tests/Spectra.py

```diff
@@ -454,8 +454,7 @@ def testWavelengthCalibrationWithRefSpec(self):
                 self.assertLess(spec.getDispRms(), self.maxRMS)
             
     def testWavelengthCalibrationWithoutRefSpec(self):
-        if True:
-            print "testing wavelength calibration"
+        if False:
```

Comment:
My understanding is that you're planning to re-activate these lines when the wavelength-per-pixel file is updated. Is that correct? If so:

- Why did you merge the new arc without the new wavelength per pixel? It seems like this should have been atomic.
- Please file an issue, triggered by the issue requesting the new wavelength-per-pixel file, to re-enable these tests;
- Please add a comment explaining what's been done.

Also, why is this in the same commit as the change above? It's a different piece of work.